definition-map.js

```diff
@@ -193,6 +193,73 @@ const MUTATIONS = {
             completedAssessmentAttemptId: "The ID of the last completed assessment attempt.",
         }
     },
+    CreateThread: {
+        metadescription: "Creates a thread of comments under a commentable type.",
+        args: {
+            forumId: "The ID of the forum.",
+            courseId: "The ID of the course.",
+            clientId: "The ID of the client.",
+            layoutId: "The ID of the layout.",
+            widgetTitle: "The title of the widget.",
+            commentableType: "The type of thread requested, based on the forum's location, e.g. discussion board, assignment, widget thread etc",
```

Comment:
Wouldn't be based on the type requested, as the user would be creating a thread, so "The type of thread to create..."